cpp/src/arrow/compute/exec/expression_benchmark.cc

```diff
@@ -119,6 +144,11 @@ BENCHMARK_CAPTURE(BindAndEvaluate, nested_array,
                   field_ref(FieldRef("struct_arr", "float")));
 BENCHMARK_CAPTURE(BindAndEvaluate, nested_scalar,
                   field_ref(FieldRef("struct_scalar", "float")));
-
+BENCHMARK_CAPTURE(BatchedExecution, execution_with_copy, expression_with_copy)
+    ->RangeMultiplier(10)
+    ->Range(1, 10000000);
+BENCHMARK_CAPTURE(BatchedExecution, execution_without_copy, expression_without_copy)
+    ->RangeMultiplier(10)
+    ->Range(1, 10000000);
```

Comment:
Add `->DenseThreadRange(1, std::thread::hardware_concurrency(), std::thread::hardware_concurrency())` to all of these benchmarks.

Reply:
Made the change.